doc/interfaces/denoise.rst

```diff
@@ -0,0 +1,90 @@
+=========
+Denoising
+=========
+
+This tutorial walks through the steps for denoising of diffusion-weighted MR images.
+There can be multiple denoising techniques.
```

Comment:
Multiple denoising methods are available in DIPY.

Reply:
Reworded in d49dc04